packages/components/src/Banner/Banner.mdx

```diff
@@ -15,6 +15,22 @@ import { Banner } from ".";
 Banners provide feedback and information about user actions and app operations.
 They're positioned at the top of the screen or near the content they reference.
 
+```ts
+import { Banner } from "@jobber/components/Banner";
+```
+
+<Playground>
+  <Banner type="success">You've successfully Hulked out</Banner>
+</Playground>
+
+## Props
+
+<Props of={Banner} />
+
+---
+
+# Usage
```

Comment:
I thought we decided not to add this as it's given that it's a usage

Reply:
Yup, I just carried that forwards into the mass edit. Fixed now.